addon/addon/models/orchestration.ts

```diff
@@ -205,3 +171,28 @@ export class OrchestrationMatrix {
     return new OrchestrationMatrix(rows, maxLength);
   }
 }
+
+export interface AnimationDefinition {
+  timeline: AnimationTimeline;
+}
+
+export type AnimationTimeline = {
+  type: 'sequence' | 'parallel';
+  animations: (MotionDefinition | AnimationTimeline)[];
```

Comment:
Interface for defining animation timelines has changed. It's slightly more verbose, but I think this is easier to use with Typescript.